build.gradle.kts

```diff
@@ -19,7 +19,7 @@ val githubUrl ="https://github.com/jeremymailen/kotlinter-gradle"
 val webUrl = "https://github.com/jeremymailen/kotlinter-gradle"
 val projectDescription = "Lint and formatting for Kotlin using ktlint with configuration-free setup on JVM and Android projects"
 
-version = "2.3.2"
+version = "2.3.3-SNAPSHOT"
```

Comment:
No need to change this, I'll update the version when I update the readme and push out the new release.